sam-styles/packages/components/button/styles/button.scss

```diff
@@ -831,4 +831,57 @@ $button-shadow: 0 0.25rem 0.5rem 0 rgba(0, 0, 0, 0.2);
     @include u-flex("column");
     flex-grow: 1;
   }
-}
\ No newline at end of file
+}
+
+.usa-button-pill {
+  @include u-radius("pill");
+  @include u-display("flex");
+  @include u-flex('align-center'); // Vertically align items in the center
+  @include u-flex('justify-center'); // Horizontally center content
+
+  svg {
+    width: 16px !important;
+    height: 16px !important;
+    fill: currentColor !important;
+    stroke-width: 1px !important;
+    stroke: color("secondary-darker");
+  }
+
+  &--next {
+    svg {
+      @include u-margin-left(1);
+      @include u-margin-top(1px);
```

Comment:
Try updating this to 3px.

At 1px:
<img width="1109" alt="Screenshot 2023-08-25 at 9 15 59 AM" src="https://github.com/GSA/sam-styles/assets/72805180/840a49b0-4ded-4bf9-8631-ad1b9b74974d">

At 3px:
<img width="1109" alt="Screenshot 2023-08-25 at 9 15 53 AM" src="https://github.com/GSA/sam-styles/assets/72805180/4a20dff8-6ab2-46ca-950d-b2507e37afe7">

Reply:
I changed it to 2px instead of 3, 3 seemed a bit too much and also uswds doesnt have a utility class for 3px margin top but they have for 2px.

Reply:
This is the alignment with 2px, it's off:
<img width="589" alt="Screenshot 2023-08-25 at 11 23 41 AM" src="https://github.com/GSA/sam-styles/assets/72805180/91ce28dc-452d-45d8-bacd-d4b4616b70c1">

> also uswds doesnt have a utility class for 3px margin top but they have for 2px.

@include u-margin-top(3px)?

Reply:
done